src/OpenLoco/Things/Misc.h

```diff
@@ -45,15 +65,70 @@ namespace OpenLoco
 
         steam_object* object() const;
 
-        static exhaust* create(loc16 loc, uint8_t type);
+        static Exhaust* create(loc16 loc, uint8_t type);
     };
 
-    struct smoke : MiscBase
+    struct MoneyEffect : MiscBase
+    {
+        uint8_t pad_20[0x2A - 0x20];
+        int32_t amount; // 0x2A - currency amount in British pounds - different currencies are probably getting recalculated
+        int8_t var_2E;  // company colour?
+        uint8_t pad_2F[0x44 - 0x2F];
+        int16_t offsetX; // 0x44
+        uint16_t wiggle; // 0x46
+
+        //static MoneyEffect* create(loc16 loc, uint8_t type);
+    };
+
+    struct VehicleCrashParticle : MiscBase
+    {
+        uint8_t pad_20[0x28 - 0x20];
+        uint16_t frame; // 0x28
+        uint8_t pad_2A[0x2E - 0x2A];
+        uint8_t colour[2];          // 0x2E
```

Comment:
may as well call the colours primary and secondary. There is `ColourScheme` in Company.h but we don't seem to be using it in many places.